libsol/transaction_summary_test.c

```diff
@@ -1,4 +1,5 @@
 #include "common_byte_strings.h"
+#include "sol/transaction_summary.h"
```

Comment:
why the `sol/` prefix?

Reply:
that header file is in different directory. I had it auto imported by my IDE as include/sol but i've changed it to the sol/... after our last discussion to make it more consistent across other files